client/app/locales/fr.json

```diff
@@ -20,8 +20,8 @@
     "modal error folder create": "Le dossier n'a pas pu être créé",
     "modal error folder exists": "Désolé, un fichier ou un dossier a déjà le même nom",
     "modal error zip empty folder": "Vous ne pouvez pas télécharger un dossier vide en tant que ZIP.",
+    "modal error updoad size exceed": "Pour des raisons de performance, vous ne pouvez pas télécharger plus de %{maxSize} fichiers en même temps. Merci de recommencer avec moins de fichiers.",
```

Comment:
I don't think we should update non-en locale without using Transifex. What do you think @m4dz and @gregorylegarec ?